subjects/DPxAI/first-hello/README.md

```diff
@@ -0,0 +1,93 @@
+## First hello
+
+> JSPowered Mode
+
+### Context
+
+Your robot winked, but you can make it talk too! Or at least, make it say his first hello to the world!
+
+> Follow the instructions, ask your peers if you are stuck and stay motivated because you are close to your goal!
+> Follow every hint you have in the subject!
+> Continue on the code from last exercise, and change the file names!
+
+### Instructions
+
+Now that you know how to make your creation move, what about making it communicate its first words to the world?
+
+After finishing these tasks, you will be able to display and hide `Hello World` in the `torso` of your robot by clicking on a second button.
+
+#### Task 1:
+
+Let's put a second button in the top right corner of the page, that will add some text when clicked. Add it in the HTML structure:
+
+```html
+<button id="speak-button">Click me to speak</button>
+```
+
+Add the button style in the CSS file:
+
+```css
+button#speak-button {
+  top: 100px;
+}
+```
+
+Also add this class to style the text we will add:
+
+```css
+.words {
+  text-align: center;
+  font-family: sans-serif;
+}
+```
+
+#### Task 2:
+
+In the JS file, like in the previous exercise, get the HTML `button` element with id `speak-button` and add an `event listener` on `click event`, triggering a `function` that will:
+
+- Select the torso element with id="torso".
+- Check if a div with the class words already exists inside the torso element.
+- If it exists, remove the existing div.
```

Comment:
Why the have to remove the element then create again? why not just hide it and show it after by appending classes?

Reply:
Because these instructions are compatible with the amount of CSS knowledge they have at this point.